modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/util/ExportImportObjectDefinitiontUtil.java

```diff
@@ -5,18 +5,54 @@
 
 package com.liferay.object.web.internal.object.definitions.portlet.action.util;
 
+import com.liferay.object.admin.rest.dto.v1_0.ObjectAction;
+import com.liferay.object.admin.rest.dto.v1_0.ObjectDefinition;
 import com.liferay.petra.function.UnsafeFunction;
 import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONFactory;
 import com.liferay.portal.kernel.json.JSONFactoryUtil;
 import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.portal.kernel.util.ListUtil;
+import com.liferay.portal.kernel.util.Validator;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.Map;
 
 /**
  * @author Gabriel Albuquerque
  */
 public class ExportImportObjectDefinitiontUtil {
 
+	public static void addActionPredefinedValues(
```

Comment:
The method's name doesn't seems appropriated because we are not only preparing the object actions, but also the object fields.

What do you think about rename it to `prepareObjectDefinition`?